api/swagger.yaml

```diff
@@ -195,6 +195,29 @@ definitions:
       Propagation:
         type: "string"
 
+  ContainerNode:
+    type: "object"
+    description: "Node container is running on in Swarm"
+    properties:
+      ID:
+        type: "string"
+      IP:
+        description: "IP Address"
+        type: "string"
```

Comment:
Can you add a `x-go-name` field (possibly we can generate the Go type from the swagger, but I think that needs a bug fix in the swagger generator; still can't hurt to have it in place already)

```yaml
        type: "string"
        x-go-name: "IPAddress"
```